src/paper/models.py

```diff
@@ -480,6 +480,16 @@ def votes_indexing(self):
             return [self.get_vote_for_index(vote) for vote in all_votes]
         return {}
 
+    def true_author_count(self):
+        registered_author_count = self.authors.count()
+        raw_author_count = len(self.raw_authors)
+
+        for author in self.raw_authors:
```

Comment:
I'm not super sure but I think `self.raw_authors` can cause an exception. It appears that we allow it to be NULL in the db. So if this value does not exist, this statement would throw a `'NoneType' is not iterable`